model/parVarSheet.h

```diff
@@ -56,12 +57,24 @@ namespace minsky
     /// column at \a x in unzoomed coordinates
     int colX(double x) const;
     /// row at \a y in unzoomed coordinates
-    int rowY(double y) const;    
+    int rowY(double y) const;
+    void moveTo(float x, float y);  
+         
+    float moveOffsX, moveOffsY,xItem,yItem;
+    ItemPtr itemFocus;        
     enum ClickType {background, internal};    
-    ClickType clickType(double x, double y) const;         
+    ClickType clickType(double x, double y);         
```

Comment:
Why is clickType not const?

Reply:
Because I initially had itemAt(x,y) inside it instead of itemFocus, but I'm changing it now...